### PR TITLE
Test edge cases

### LINKED ISSUE #81: Test rare flush error
https://github.com/Level/level-ws/blob/182314539e46f1b0c056a622c676612ad005b447/level-ws.js#L67-L70

Line 69 is not covered atm.

### PR DESCRIPTION
**Pull request by vweevers**
Closes #81 and a few more edge cases. 100% coverage!

### CODE REVIEW

**Comment by vweevers** [comment]
`Could not resolve host: nodejs.org` again. Retries are not helping, waiting a little bit.

**Comment by vweevers** [comment]
Code coverage is immensely helpful in triggering these edge cases.

File: test.js [hunk 1]
```diff
@@ -17,15 +17,18 @@ function monitor (stream) {
   return order
 }
 
-function slowdown (db) {
+function monkeyBatch (db, fn) {
```
Comment (ralphtheninja) [inline test.js]: Funny! :)
